source/board/lpc55s26_bl.c

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file    lpc5516_bl.c
```

Comment:
It should be `lpc5526_bl.c`

Reply:
Unless you object, we've renamed it to lpc55s26_bl.c instead to be consistent with NXP's SDK (NXP filenames were named S26 for both S26 and the 26 variants).